app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -210,19 +203,44 @@ export default function ManageChannelMembers({
         setChannelMembers(clone);
     }, [channelMembers]);
 
+    const sortedProfiles = useMemo(() => [...profiles].sort((a, b) => {
```

Comment:
The first list of profiles is sorted by admin, so all admins come first. This makes sure that the "temporal result" is sorted the same way as the actual search result.